packages/shopware-6-client/src/interfaces/response/Error.ts

```diff
@@ -0,0 +1,19 @@
+interface Error {
+  status: string;
+  code: string;
+  title: string;
+  detail: string;
+  meta: any;
+  trace?: Array<{
+    file: string;
+    line: number;
+    function: string;
+    class: string;
+    type: string;
+    args: Array<any>;
```

Comment:
isn't it a string array?
```suggestion
    args: any[];
```